packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -433,7 +433,13 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
                 isLoading={isLoading || isLoadingExplore}
                 extraMenuItems={
                     savedChartUuid !== null &&
-                    user.data?.ability?.can('manage', 'Explore') && (
+                    user.data?.ability?.can(
+                        'manage',
+                        subject('Explore', {
+                            organizationUuid: savedChartUuid,
```

Comment:
this is wrong , is comparing organizationUuid and  savedChartUuid

Reply:
oops. My bad.